src/Microsoft.VisualStudio.Sdk.TestFramework.Xunit/BrokeredServiceContractTestBase`2.cs

```diff
@@ -79,6 +78,11 @@ public BrokeredServiceContractTestBase(ITestOutputHelper logger, ServiceRpcDescr
     /// </summary>
     protected SourceLevels DescriptorLoggingVerbosity { get; set; } = SourceLevels.Verbose;
 
+    /// <summary>
+    /// Gets the test count.
+    /// </summary>
+    protected int TestCounter => testCounter;
```

Comment:
Why was this necessary? It doesn't appear to be used anywhere.

Reply:
Artifact of a previous iteration, thanks for catching that. I'll remove it.